update explorer content and imgs (#5773)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/explore-multiple-projects.md

description: "Learn about project-level lineage in dbt Explorer and its uses."
---

View all the projects and public models in your account (where public models are defined) and gain a better understanding of your cross-project resources and how they're used.

> The resource-level lineage graph for a project displays the cross-project relationships in the DAG, with an icon indicating whether  if it's a project resource. Notice the **PRJ** icon in the left side of the node name.

Reviewer: sugg: for slightly better clarity 

```suggestion
The resource-level lineage graph for a project displays the cross-project relationships in the DAG, with a **PRJ** icon indicating whether or not it's a project resource. That icon is located to the left side of the node name.
```

To view the project-level lineage graph, click the **View lineage** icon in the upper right corner from the main overview page:
- This view displays all the projects in your account and their relationships.
- Viewing an upstream (parent) project displays the downstream (child) projects that depend on it.
- Selecting a model reveals its dependent projects in the lineage.
- Click on an upstream (parent) project to view the other projects that reference it in the **Relationships** tab, showing the number of downstream (child) projects that depend on them. 

>   - This include all projects listing the upstream one as a dependency in its `dependencies.yml` file, even without a direct `{{ ref() }}`.

Reviewer: ```suggestion
  - This includes all projects listing the upstream one as a dependency in its `dependencies.yml` file, even without a direct `{{ ref() }}`.
```

>   - Selecting a project node from a public model opens its detailed lineage graph, subject to your [permission](/docs/cloud/manage-access/enterprise-permissions).

Reviewer: ```suggestion
  - Selecting a project node from a public model opens its detailed lineage graph if you have the [permissions](/docs/cloud/manage-access/enterprise-permissions) to do so.
```

> <Lightbox src="/img/docs/collaborate/dbt-explorer/cross-project-lineage-parent.png" width="100%" height="100" title="View your cross-project lineage in a parent project and the other projects that reference it by clicking the 'Relationships' tab."/>

Reviewer: ```suggestion
<Lightbox src="/img/docs/collaborate/dbt-explorer/cross-project-lineage-parent.png" width="100%" height="100" title="View your cross-project lineage in a parent project and the other projects that reference it by clicking the "Relationships" tab."/>
```

Author: the build is failing bc of the quotes. i think this is why i iniitally use apostrophes in the captions. going to go back to apostrophe's

> <Lightbox src="/img/docs/collaborate/dbt-explorer/cross-project-lineage-parent.png" width="100%" height="100" title="View your cross-project lineage in a parent project and the other projects that reference it by clicking the "Relationships" tab."/>

Author: ```suggestion
<Lightbox src="/img/docs/collaborate/dbt-explorer/cross-project-lineage-parent.png" width="100%" height="100" title="View your cross-project lineage in a parent project and the other projects that reference it by clicking the 'Relationships' tab."/>
```

When viewing a downstream (child) project that imports and refs public models from upstream (parent) projects:
- Public models will show up in the lineage graph and you can click on them to view the model details.
- Clicking on a model opens a side panel containing general information about the model, such as the specific dbt Cloud project that produces that model, description, package, and more.

> - Double-clicking on a model from another project opens the resource-level lineage graph of the parent project, which is subject to your permissions.

Reviewer: ```suggestion
- Double-clicking on a model from another project opens the resource-level lineage graph of the parent project, if you have the permissions to do so.
```

> <Lightbox src="/img/docs/collaborate/dbt-explorer/cross-project-child.png" width="100%" height="100" title="View a downstream 'child' project that importants and refs public models from the upstream 'parent' project."/>

Reviewer: ```suggestion
<Lightbox src="/img/docs/collaborate/dbt-explorer/cross-project-child.png" width="100%" height="100" title="View a downstream (child) project that importants and refs public models from the upstream (parent) project."/>
```

## Explore the project-level lineage graph

For cross-project collaboration, you can interact with the DAG in all the same ways as described in [Explore your project's lineage](/docs/collaborate/explore-projects#project-lineage) but you can also interact with it at the project level and view the details. 

To get a list view of all the projects in your account:
- Navigate to the top left section of the **Explore** page, near the navigation bar.
- Hover over the project name and select the account name.

> - This will return a project view page, allowing you to see all the projects in the account. It returns a public model list, project list, and a search bar for project searches.

Reviewer: not sure what "this" is referring to here in this bullet. sugg improving for clarity

Author: changed it to:

< - Hover over the project name and select the account name to return a project view page, allowing you to see all the projects in the account.

> - It returns a public model list, project list, and a search bar for project searches.

Reviewer: similar to before, not sure what "it" is referring to here: 

"To get a list view of all the projects in your account, it returns a public model list, project list, and a search bar for project searches" 

maybe the "original sentence" just needed to be indented for an L2 bullet list (instead of L1)

Author: ok got it! updated to the below. 

> To get a list view of all the projects in your account:
> - Navigate to the top left section of the **Explore** page, near the navigation bar.
> - Hover over the project name and select the account name. This takes you to a project view page, where you can view all the projects in the account.
> - Click the **List view** icon in the page's upper right corner to see a list view of all the projects in the account.
> - The list view page displays a public model list, project list, and a search bar for project searches.
> - Click the **Lineage view** icon in the page's upper right corner to view the project-level lineage gra

> - You can also view the project-level lineage graph by clicking the **Lineage view** icon in the page's upper right corner.

Reviewer: sugg: parallel construction of bulleted list for consistency/readability

```suggestion
- Click the **Lineage view** icon in the page's upper right corner to view the project-level lineage graph.
```
